Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->